protos/google/cloud/storage/tests/CMakeLists.txt

```diff
@@ -0,0 +1,49 @@
+# Copyright 2024 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may not
+# use this file except in compliance with the License. You may obtain a copy of
+# the License at
+#
+# https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations under
+# the License.
+
+# Signed URL conformance tests are parsed using protos. In order to simplify the
+# build files, only build these conformance tests if gRPC in GCS is compiled.
+if ((NOT GOOGLE_CLOUD_CPP_STORAGE_ENABLE_GRPC) OR (NOT BUILD_TESTING))
+    return()
+endif ()
+
+include(GoogleCloudCppCommon)
+find_package(Protobuf CONFIG QUIET)
+if (NOT Protobuf_FOUND)
+    find_package(Protobuf REQUIRED)
+endif ()
```

Comment:
Why not just `find_package(Protobuf REQUIRED)` ?

Reply:
Because CMake <= 3.xx searches for non-CONFIG first, and we prefer to use the `CONFIG` version if available.